apps/next/src/styles.scss

```diff
@@ -26,16 +27,16 @@ html {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   text-rendering: optimizeLegibility;
-  font-size: 16px;
+  font-size: $fluid-spacing--medium;
```

Comment:
No, please don't use spacing tokens to assign font sizes, just because their units match by happenstance. 
There are tokens for body text, please use these.